CHANGELOG.md

```diff
@@ -1,5 +1,25 @@
 ## Next release
 
+## 0.42.0 / 2020-09-09
+
+Most importantly the Prometheus Operator now lives in its own indepent GitHub organization.
```

Comment:
It had already happened for 0.41?

Reply:
I don't think so. All PRs about moving were between [v0.41.1...master](https://github.com/prometheus-operator/prometheus-operator/compare/v0.41.1...master).